microsoft/testsuites/core/provisioning.py

```diff
@@ -191,6 +193,31 @@ def verify_reboot_in_platform(
             case_name="verify_reboot_in_platform",
         )
 
+    @TestCaseMetadata(
+        description="""
+        This case runs smoke test on a node provisioned with ultra disk.
+        The test steps are same as `smoke_test`.
+        """,
+        priority=3,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                data_disk_type=schema.DiskType.UltraSSDLRS,
+                os_disk_type=schema.DiskType.StandardHDDLRS,
+                data_disk_iops=search_space.IntRange(min=160000),
```

Comment:
Do we need this high of an IOPS requirement? Do we need any IOPS requirement?

This will boot with 1TB disk.

Reply:
160,000 IOPS is the max supported by a single ultradisk. Lets use it.

Reply:
It's a provisioning test case, it doesn't need to be max value.

Reply:
What would be a good size to use then?

Reply:
It doesn't need to set iops requirements on provisioning test case.

Reply:
Fixed it. Removed the IOPS requirement.